utils/httpbin/httpbin.go

```diff
@@ -24,6 +24,7 @@ func Handler() http.Handler {
 	m.HandleFunc("/delay/", delayHandler)
 	m.HandleFunc("/status/", statusHandler)
 	m.HandleFunc("/stream-bytes/", streamBytesHandler)
+	m.HandleFunc("/stream-discard/", streamDiscardHandler)
```

Comment:
stream comes from [Dynamic data](https://httpbin.org/#/Dynamic%20data) Generates random and dynamic data

It's more like sink just read all body, it feels like any function should do it.
You could be more creative and have count-bytes that one would return body size as a header etc.